AirCasting/SessionViews/SessionCartView.swift

```diff
@@ -107,7 +128,13 @@ private extension SessionCartView {
     }
     
     func displayButtons(threshold: SensorThreshold) -> some View {
-        HStack(spacing: 20){
+        HStack(spacing: 20) {
+            if isFollowing && !(session.type == .mobile) {
```

Comment:
I think `isFollowing` would be enough as we only set "isFollowing" on fixed sessions. But if you want to stick with the second part, why do you use !`(session.type == .mobile)` instead of `(session.type == .fixed)`?

Reply:
Yes, you are right. It was for nothing. Thanks!